### PR TITLE
SHIPPING-2200 Add delete pickup methods spec

### PR DESCRIPTION
**Pull request by bc-luke**
Add Open API documentation for [new delete pickup methods operation](https://github.com/bigcommerce/api-proxy-java/pull/1544).

⚠️ This API is not generally available, so the documentation shouldn't be made public yet. It'll first be part of a beta.

@bigcommerce/dev-docs

### CODE REVIEW

**Review by agchan12 — APPROVED** [review]

File: reference/pickup.v3.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+openapi: 3.0.1
+info:
+  title: Pickup
```
Comment (bc-andreadao) [inline reference/pickup.v3.yaml:3]:
```suggestion
  title: Pickup Methods
```

File: reference/pickup.v3.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      store_hash:
+        default: store_hash
+paths:
+  '/stores/{store_hash}/v3/pickup/methods':
+    delete:
+      tags:
+        - Pickup Methods
+      description: Delete existing pickup methods.
+      parameters:
+        - name: id:in
```
Comment (bc-andreadao) [inline reference/pickup.v3.yaml:18]: id?

**Comment by slsriehl** [comment]
Corollary devdocs ticket [DEVDOCS-3541](https://jira.bigcommerce.com/browse/DEVDOCS-3541).

cc @bc-andreadao

**Comment by agchan12** [comment]
Closing this PR as the work has been merged in here https://github.com/bigcommerce/api-beta-multi-location-inventory/pull/7

**Pull request closed without merging**